src/main/resources/avro/readmethods.avdl

```diff
@@ -14,8 +14,10 @@ to `ReferenceSet`s containing that same `Reference`. If no reference is
 specified, all `ReadGroup`s must be aligned to the same `ReferenceSet`.
 */
 record SearchReadsRequest {
-  /** If nonempty, restrict this query to reads within the given readgroups. */
-  array<string> readGroupIds = [];
+  /**
+  The ReadGroup to search.
+  */
+  string readGroupId;
```

Comment:
Multi-BAM slicing is a very useful operation that server can optimize but client cannot. In principle, a right backend could merge 1,000 related samples efficiently on the server side. Do we want a client to open 1,000 connections in parallel just to get data in a tiny region? Or are we thinking that we don't need to support this operation because our BAM-based backend cannot achieve it?